internal/test/testdata/verify/cases/case_010.yaml.input

```diff
@@ -0,0 +1,59 @@
+-- testdata/principals.yaml --
+---
+principals:
+  john:
+    policyVersion: '20210210'
+    id: john
+    roles:
+      - employee
+    attr:
+      department: marketing
+      geography: GB
+      team: design
+
+-- testdata/resources.yaml --
+---
+resources:
+  john_leave_request:
+    policyVersion: '20210210'
+    kind: leave_request
+    id: XX125
+    attr: &attr
+      department: marketing
+      geography: GB
+      id: XX125
+      owner: john
+      team: design
+
+-- suite_test.yaml --
+---
+name: TestSuite
+description: Tests
+tests:
+  - name: Test 1
+    input:
+      principals:
+        - john
+      resources:
+        - john_leave_request
+      actions:
+        - view:public
+    expected:
+      - principal: john
+        resource: john_leave_request
+        actions:
+          view:public: EFFECT_ALLOW
+
+  - name: Test 1
+    input:
+      principals:
+        - john
```

Comment:
If we change the principal here to someone else, would it still be detected as a duplicate test? We should probably disallow two test cases to have the same name regardless of whether they exercise completely different principals/resources/actions.

Reply:
I modified the relevant part. Now, we;
- continue even if we have the same `principal|resource|action` pair in two test cases. (`case_08`)
- continue even if we have the same `principal|resource|action` pair with different aux data. (`case_09`)
- fail the test if there are any duplicate test names present. (`case_10`)

Does this sound good?